pyxodr/road_objects/road.py

```diff
@@ -84,7 +105,10 @@ def traffic_orientation(self) -> TrafficOrientation:
         return traffic_orientation
 
     def __getitem__(self, name):
-        return self.road_xml.attrib[name]
+        try:
+            return self.road_xml.attrib[name]
+        except KeyError:
+            return None
```

Comment:
Returning `None` here causes the tests to fail because in [road.py](https://github.com/driskai/pyxodr/blob/1095d123de302f0199b2e68faff9687b28555fc5/pyxodr/road_objects/road.py#L73) we set a default value when an `KeyError` is thrown. Probably it's best to add the try block in the function that gets the attribute value.